servers/rendering/renderer_viewport.cpp

```diff
@@ -828,16 +817,15 @@ uint32_t RendererViewport::Viewport::get_view_count() {
 	return view_count;
 }
 
-void RendererViewport::viewport_set_size(RID p_viewport, int p_width, int p_height) {
+void RendererViewport::viewport_set_size(RID p_viewport, int p_width, int p_height, uint32_t p_view_count) {
 	ERR_FAIL_COND(p_width < 0 && p_height < 0);
 
 	Viewport *viewport = viewport_owner.get_or_null(p_viewport);
 	ERR_FAIL_COND(!viewport);
 
 	viewport->size = Size2(p_width, p_height);
 
-	uint32_t view_count = viewport->get_view_count();
-	RSG::texture_storage->render_target_set_size(viewport->render_target, p_width, p_height, view_count);
+	RSG::texture_storage->render_target_set_size(viewport->render_target, p_width, p_height, p_view_count);
```

Comment:
I think we need to store view_count in our `Viewport` struct as well and make sure that other places where render_target_set_size is called, we use our variable.

Reply:
Ok! I already had that in my other PR, I'd be happy to bring that over as well